ports/heroes/Heroes.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
```

Comment:
Missing space between shebang and `XDG_DATA_HOME` bothers me so much more than it should.
```suggestion

XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
```